nixos/modules/services/system/dbus.nix

```
@@ -57,12 +50,29 @@ in
           <filename><replaceable>pkg</replaceable>/share/dbus-1/services</filename>
         '';
       };
+
+      socketActivated = mkOption {
+        type = types.nullOr types.bool;
+        default = null;
+        visible = false;
+        description = ''
+          Removed option, do not use.
+        '';
+      };
     };
   };
 
   ###### implementation
 
   config = mkIf cfg.enable {
+    warnings = optional (cfg.socketActivated != null) (
+      let
+        files = showFiles options.services.dbus.socketActivated.files;
+      in
+        "The option 'services.dbus.socketActivated' in ${files} no longer has"
+        + " any effect. The user D-Bus session is now always socket activated"
+        + " and can safely be removed."
```

```suggestion
        + " any effect and can be safely removed: the user D-Bus session is"
        + " now always socket activated."
```

"is now always socket activated and can safely be removed" feels non-sequitur (the "safely be removed" does not apply to the "user D-Bus session", but instead to the option `services.dbus.socketActivated`).

---

Thanks! Applied.